### DISCUSSION TITLE
Autenticação e Autorização [BANCO]

### DISCUSSION THREAD

**Discussion by eoliverdev** [balta-io/forum#338] [Tira Dúvidas] [answered]
Opa Balta, blz?

No Módulo "Fundamentos do ASP.NET 6" em Autenticação e Autorização, você comenta sobre um arquivo .sql pra popular o banco, que você não diz onde está pra pegarmos e o que você mostra no vídeo está na sua máquina. Procurei no código fonte e não achei. Pode me ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @eoliverdev, como vai?

Nos módulos, tem uma pasta Scripts:
https://github.com/balta-io/2811/tree/main/Modulo%206/Scripts

Os arquivos de banco ficam lá...
Nos módulos onde usamos os mesmos bancos dos anteriores, não tem esta pasta por que imagino que já esteja com banco preparado do módulo anterior.
